### ISSUE TITLE
Feedback count should be public + private

### ISSUE THREAD

**Issue by lenglund** [ubc/compair#546]
Right now, it seems that the student indicator of feedback (displayed by assignment on the course screen) only relates to private feedback. For consistency, this should reflect what we actually show under the student's answer as replies, which is both private and public feedback.

**Comment by andrew-gardener** [comment]
If you mean the `# peer(s) left you feedback` label it actually is the number of unique users that left feedback. This includes instructors, teaching assistants, other students and yourself (if left self-evaluation or private/public comment).

Should we instead reword it to something like `you have been left feedback # time(s)` and it be equal to the total (public+private)?

**Comment by andrew-gardener** [comment]
actually scratch that, I'm not sure what's happening with the count but it is definitely wrong (will look further into)

**Comment by lenglund** [comment]
Can we do "# feedback items received"?
